provider/resources.go

```diff
@@ -199,4 +201,16 @@ func Provider() tfbridge.ProviderInfo {
 			Namespaces: namespaceMap,
 		},
 	}
+
+	err := x.ComputeDefaults(&prov, x.TokensKnownModules("f5bigip_", mainMod, []string{
+		"CM_",
+		"Ltm_",
+		"Net_",
+		"Sys_",
+		"Ssl_",
+		"VCMP_",
```

Comment:
This doesn't work. These need to be the prefixes found in TF tokens, not the pulumi modules. This should be `"cm_"`, `"ltm_"`, etc.

If you're trying to handle non-standard mappings like `"cm_" -> "CM"`, you need to do that in your token factory. [Here](https://github.com/pulumi/pulumi-alicloud/blob/1df9eca232a8af1f8d18d1ba7ae6b538b60d34b1/provider/resources.go#L2402-L2435) is pulumi-alicloud's code to accomplish custom mappings.

The API isn't great for this use case, and suggestions for improvement would be welcome.